packages/gateway/index.ts

```
@@ -0,0 +1,19 @@
+import { createServer } from 'node:http';
+import { createGatewayRuntime } from '@graphql-hive/gateway-runtime';
+import http from '@graphql-mesh/transport-http';
+import { env } from '../server/src/environment';
```

ESLint

[error] Missing file extension for "../server/src/environment"

(import/extensions)